README.md

```diff
@@ -37,27 +42,32 @@ The `install-dev.sh` script will build and push an image of the Tekton Dashboard
 2. [Checkout](https://github.com/tektoncd/dashboard/blob/master/DEVELOPMENT.md#checkout-your-fork) the repository
 
 If you want to install the dashboard into the tekton-pipelines namespace:
+
 - Install the Dashboard `./minishift-install-dashboard.sh`
 
 If you want to install the dashboard into any other namespace:
+
 - Install the Dashboard `./minishift-install-dashboard.sh -n {NAMESPACE}`
 
 3. Wait until the pod `tekton-dashboard-1` is running in the namespace the Dashboard is installed into
 
 ## Accessing the Dashboard on Minishift
-The Dashboard can be accessed by running `kubectl port-forward $(kubectl get pod --namespace tekton-pipelines -l app=tekton-dashboard -o name)  --namespace tekton-pipelines 9097:9097` 
-If installed into a namespace other than tekton-pipelines then the dashboard can be accessed by running `kubectl port-forward $(kubectl get pod --namespace NAMESPACE -l app=tekton-dashboard -o name)  --namespace NAMESPACE 9097:9097`
+
+The Dashboard can be accessed by running `kubectl --namespace tekton-pipelines port-forward svc/tekton-dashboard 9097:9097`
+If installed into a namespace other than tekton-pipelines then the dashboard can be accessed by running `kubectl --namespace $NAMESPACE port-forward svc/tekton-dashboard 9097:9097`
 You can access the web UI at `http://localhost:9097/`
 
 ## Uninstalling the Dashboard on Minishift
+
 The Dashboard can be uninstalled on Minishift by running the command `./minishift-delete-dashboard.sh` Use `-n {NAMESPACE}` on the end of the command if installed into a namespace other than `tekton-pipelines`
 
 ## Accessing the Dashboard
-The Dashboard can be accessed through its ClusterIP Service by running `kubectl proxy`. Assuming tekton-pipelines is the install namespace for the dashboard, you can access the web UI at `localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/`
+
+The Dashboard can be accessed through its ClusterIP Service by running `kubectl proxy`. Assuming tekton-pipelines is the install namespace for the dashboard, you can access the web UI at `localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/`. An alternativ way to access the Dashboard is using `kubeectl port-forward` e.g. if you installed the Tekton Dashboad into the `tekton-pipelines` namespace (which is the deafult) you can access the Dashboard with `kubectl --namespace tekton-pipelines port-forward svc/tekton-dashboard 9097:9097`
```

Comment:
`kubeectl` -> `kubectl` 
`alternativ` -> `alternative`

Reply:
Thanks for the catch!